docs/interactions/Message_Components.md

```diff
@@ -1,14 +1,38 @@
 # Message Components
 
-Message components--we'll call them "components" moving forward--are a framework for adding interactive elements to the messages your app or bot sends. They're accessible, customizable, and easy to use.
+Message components&#x2014;we'll call them "components" moving forward&#x2014;are a framework for adding interactive elements to the messages your app or bot sends. They're accessible, customizable, and easy to use.
 
 There are several different types of components; this documentation will outline the basics of this new framework and each example.
 
-## What are Components
+## Components
 
-Components are a new field on the [message object](#DOCS_RESOURCES_CHANNEL/message-object), so you can use them whether you're sending messages or responding to a [slash command](#DOCS_INTERACTIONS_SLASH_COMMANDS/) or other interaction.
+Components are a new field on the [message object](#DOCS_RESOURCES_CHANNEL/message-object), so you can use them whether you're sending messages or responding to a [slash command](#DOCS_INTERACTIONS_SLASH_COMMANDS) or other interaction.
```

Comment:
pretty sure page links always have a trailing slash, are you sure they work without it?

Reply:
I did a bit of digging to see and came across this in [Community Resources/Dispatch Tools](https://discord.com/developers/docs/topics/community-resources#dispatch-tools):
![image](https://user-images.githubusercontent.com/43933794/122700083-3bd42280-d210-11eb-832f-04e1e5080314.png)
Clicking on it seems to redirect properly, although it seems like the majority of the docs use a trailing slash (ironically this PR is about consistency lol). I am a bit curious to see what typpo or night think about trailing slashes though.